app/src/main/java/io/halkyon/services/ClaimService.java

```diff
@@ -75,9 +75,7 @@ public void setClaimStatusAndAttempts(Claim claim) {
                 claim.persist();
                 return;
             }
-        }
-
-        if (claim.service != null) {
+        } else {
```

Comment:
This is the line that is causing some tests to fail.
This is because a few lines above, the claim is setting the service and then it's expected to continue over here.